guides/doc-Upgrading_and_Updating/topics/upgrading_content_hosts.adoc

```diff
@@ -41,7 +41,8 @@ ifdef::satellite[]
 * If your deployment uses `katello-agent` and *goferd*, enter `katello-agent`.
 * If your deployment does not use `katello-agent` and *goferd*, enter `katello-host-tools`.
 +
-. Until https://bugzilla.redhat.com/show_bug.cgi?id=1649764[BZ#1649764] is resolved, from the *Update* list, you must select *via remote execution*.
+. https://bugzilla.redhat.com/show_bug.cgi?id=1649764[BZ#1649764] is closed as WONTFIX.
+To workaround the problem, from the *Update* list, you must select *via remote execution*.
```

Comment:
```suggestion
. Select *via remote execution* from the *Update* list.
```
IMHO you can drop the mention to BZ completely. Also please drop "To work around the problem" since no problem was introduced prior to this sentence.